### ISSUE TITLE
The custom domain cloudfront.net giving 404

### ISSUE THREAD

**Issue by kunal-supe** [aws-amplify/amplify-console#1484]
**App Id**: d3dy4an9tja241
**Region**:
**Step**:  The custom domain is added, it says need to wait until the <id>.cloudfront.net id migrated. But its giving 404 and hence the domain is not working

**Status**: Available (but not working)

**Issue/question**
After adding the custom domain, the cloudfront.net and parked domain isn't working after status: Available

**Comment by semcneil** [comment]
I am getting this same problem. The cloudfront.net URL never stops giving a 404 error, even after days of waiting. SSL creation and configuration have both finished successfully. I tried setting my domain's CNAME to point towards the amplifyapp.com URL and it works. I have tried deleting and restarting the custom domain, and even on a different project, but all the cloudfront.net URLs give a 404 error.
